src/pages/Auth/Auth.module.scss

```diff
@@ -0,0 +1,61 @@
+* {
+	box-sizing: border-box;
+}
+
+.authorization {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	height: 77vh;
+}
+
+.error {
+	color: red;
+	margin-bottom: 20px;
+}
+
+.form {
+	width: 270px;
+	padding: 10px 20px;
+	display: flex;
+	flex-direction: column;
+	gap: 18px;
+
+	.formLabel {
```

Comment:
make sure your teammates uses the same scss benefits in the same situations. your codestyle should align with everyone else